### ISSUE TITLE
[BUG] Confusion about Config Documentation

### ISSUE THREAD

**Issue by besendorf** [dillonzq/LoveIt#674]
### Describe the bug 描述你遇到的错误

Im a bit confused about the config part of the [documentation](https://hugoloveit.com/theme-documentation-basics). In the [2.3 Basic Configuration](https://hugoloveit.com/theme-documentation-basics/#basic-configuration) Section a example configuration is provided inlcuding baseurl, menu and markup configuration.
In [3.1 Site Configuration](https://hugoloveit.com/theme-documentation-basics/#site-configuration) there is another example configuartion, however it lacks some of the entries from the first config like the baseurl and overlaps with otherconfigurations like the markup configuration. 
So now I am confused whether I should remove the first configuration file and use the second one instead or if I should merge them.
I think the documentation should be improved with configs that are either disjoint so one can just append the one to the other or the second config should include the first one. Or just only use 1 config file.

**Comment by andatoshiki** [comment]
Hi, the configuration file of 2.3 Basic Configuration you referred from the docs is a complete and full basic **EXAMPLE** `toml` file for launching your sites, it includes all the parameters of your blog site which includes the `exampleSite="https://xxx.xxx"` as you described yet it's just a **reference config file**. But in 3.1 Site configuration it only specifies **some** specific site parameter and entries which is an extent from 2.3, the author wrote the doc more detailly so that users are able to customize the theme more flexibly.

In short, to ensure your site launches without errors, you should copy the complete config `toml` file content in the 2.3 Basic configuration to your local theme `config.toml`, and read through 3.1 Site configuration for personalizing your site.
